cardano-tx-generator/cardano-tx-generator.cabal

```diff
@@ -110,23 +110,3 @@ executable cardano-tx-generator
                      , transformers-except
 
   default-extensions:  NoImplicitPrelude
-
-test-suite cardano-tx-generator-test
-  hs-source-dirs:       test
-  main-is:              Main.hs
-  type:                 exitcode-stdio-1.0
-
-  build-depends:        base >=4.12 && <5
-                      , tasty-hunit
-                      , tasty
```

Comment:
Lets keep the test cases.
I find them useful (and I actually added new tests in my last PR).
Even if it was only for having a stub of a test suite, it should stay
If the break nix in a complicated way then lets disable them and resurrect soon.

Reply:
Ok, agreed.